modules/sso.php

```diff
@@ -305,20 +322,6 @@ public function validate_settings_remove_login_form_checkbox( $input ) {
 	}
 
 	/**
-	 * Removes 'Lost your password?' text from the login form if user
```

Comment:
We're able to remove this because we now *always* remove the "Lost your password?" link when showing the WPCOM SSO form.